napari/layers/shapes/_tests/test_shapes_mouse_bindings.py

```diff
@@ -499,48 +393,32 @@ def test_drag_shape(create_known_shapes_layer, Event):
     assert value == (0, None)
```

Comment:
Test is failing here.
```
  >       assert value == (0, None)
  E       assert (0, 7) == (0, None)
  E         At index 1 diff: 7 != None
  E         Full diff:
  E         - (0, None)
  E         + (0, 7)
```

Reply:
This is actually an issue I was having when debugging #5802. IIRC this test is actually wrong, and needed to be fixed. I'd liek to first merge that, and then update tests here so I don't have to do the work twice :)

Reply:
Then at a `@pytest.mark.knownfail` to the test , or comment this assert ? 

I don't think anyone will merge if all the tests are red...

Reply:
Sorry, I wasn't clear: I want to first merge #5802, and then deal with this test in this PR (so no rush in merging this for now).

Reply:
Ah, let me convert to draft then maybe so that people sip reviewing it ?

Reply:
Good idea, thanks!